Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Blobs"
 uuid = "2063b8be-a4ad-11e8-1fb9-93d01944c8ed"
 authors = ["Jamie Brandon <jamie@scattered-thoughts.net>"]
```

Comment:
Haha, just to not review without leaving a comment — do we want to keep this in?

Reply:
🤷  It's maybe nice to recognize the work he did early on it?
But also i think it's fine to delete it yeah.